CHANGELOG.md

```diff
@@ -49,6 +49,7 @@
 
 ## Changes
 
+- Directories are now printed with an additional path separator at the end: `foo/bar/`
```

Comment:
I have released a new version in the meantime, so it would be great if we could move this line up into the "unreleased" section.